applications/ConstitutiveLawsApplication/custom_processes/set_automated_initial_damage_process.cpp

```diff
@@ -0,0 +1,133 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: kratos/license.txt
```

Comment:
```suggestion
//  License:               BSD License
//                              license: kratos/license.txt
```
The same for the others headers